viewer/packages/camera-manager/src/Flexible/IFlexibleCameraManager.ts

```diff
@@ -49,4 +49,10 @@ export interface IFlexibleCameraManager extends CameraManager {
    * @param callback - The callback function to be removed from the controls type change listeners.
    */
   removeControlsTypeChangeListener(callback: FlexibleControlsTypeChangeDelegate): void;
+
+  /**
+   * Updates the inner bounding box of the camera manager used for camera sensibility
+   * @param innerBoundingBox - The new inner bounding box.
+   */
+  updateInnerBoundingBox(innerBoundingBox: Box3): void;
```

Comment:
This does not need to be exposed on the interface. It's only intended for internal use

Reply:
Fixed